MastodonSDK/Sources/MastodonCore/Extension/CoreDataStack/Instance.swift

```diff
@@ -24,9 +24,43 @@ extension Instance {
     }
 }
 
+extension Instance {
+    public var configurationV2: Mastodon.Entity.V2.Instance.Configuration? {
+        guard let configurationRaw = configurationV2Raw else { return nil }
+        guard let configuration = try? JSONDecoder().decode(Mastodon.Entity.V2.Instance.Configuration.self, from: configurationRaw) else {
+            return nil
+        }
+
+        return configuration
+    }
+    
+    static func encodeV2(configuration: Mastodon.Entity.V2.Instance.Configuration) -> Data? {
+        return try? JSONEncoder().encode(configuration)
+    }
+}
+
 extension Instance {
     public var canFollowTags: Bool {
-        guard let majorVersionString = version?.split(separator: ".").first else { return false }
-        return Int(majorVersionString) == 4 // following Tags is support beginning with Mastodon v4.0.0
+        version?.majorServerVersion(greaterThanOrEquals: 4) ?? false // following Tags is support beginning with Mastodon v4.0.0
+    }
+}
+
+extension String {
+    public func majorServerVersion(greaterThanOrEquals comparedVersion: Int) -> Bool {
+        guard
+            let majorVersionString = split(separator: ".").first,
+            let majorVersionInt = Int(majorVersionString)
+        else { return false }
+        
+        return majorVersionInt >= comparedVersion
+    }
+}
+
+extension Instance {
+    var isTranslationEnabled: Bool {
+        if let configuration = configurationV2 {
+            return configuration.translation?.enabled == true
+        }
+        return false
```

Comment:
Maybe it makes sense to group `canFollowTags` and `isTranslationsEnabled` into one extension?